blaze/server/server.py

```diff
@@ -279,6 +279,11 @@ def compserver(dataset):
 
     try:
         result = compute(expr, {leaf: dataset})
+
+        if iscollection(expr.dshape):
+            result = into(list, result)
```

Comment:
I can change this; however, I was just moving this code up into the try/except block. The `into` was the existing code.